### ISSUE TITLE
why deepfm use reduce_sum in loss?

### ISSUE THREAD

**Issue by chengmengli06** [PaddlePaddle/models#4895]
why deepfm use reduce_sum in loss?  The loss is large, while the model could converge, it is strange. 

https://github.com/PaddlePaddle/models/blob/release/1.8/PaddleRec/ctr/deepfm_dygraph/train.py#L108

**Comment by hutuxian** [comment]
Generally speaking, both `reduce_sum` and `reduce_mean` are OK, because you can tune the learning-rate (a multipler of batch_size) to make them same.

**Comment by chengmengli06** [comment]
It should be the same, but when I change it to reduce_mean, the auc drops by 1 percent to 0.793, so what happened? tune learning-rate does not improve. Adam is not sensitive to scale of gradients.

**Comment by hutuxian** [comment]
Sorry for late reply. For normal SGD it is the same, that't why i say `Generally speaking`. For Adam, i think other hyper-parameter should be tuned if you change the network.
